test/framework/releaseVersions.go

```diff
@@ -60,5 +74,29 @@ func (e *ClusterE2ETest) GetLatestMinorReleaseFromReleaseBranch(releaseBranchVer
 		e.T.Fatalf("Releases manifest doesn't contain a version of the previous minor release")
 	}
 
-	return latestPrevMinorRelease
+	binaryPath, err := e.getBinary(latestPrevMinorRelease)
+	if err != nil {
+		e.T.Fatalf("Failed getting binary for latest previous minor release: %s", err)
+	}
+
+	return binaryPath
+}
+
+func (e *ClusterE2ETest) getBinary(release *releasev1alpha1.EksARelease) (string, error) {
+	latestReleaseBinaryFolder := filepath.Join("bin", release.Version)
```

Comment:
so the directory will be in the format `./bin/v0.6.0/`?

Reply:
yes!